integration-tests/narayana-jta/src/main/java/io/quarkus/narayana/jta/TransactionScopedBean.java

```diff
@@ -1,9 +1,17 @@
 package io.quarkus.narayana.jta;
 
+import java.util.concurrent.atomic.AtomicInteger;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
 import javax.transaction.TransactionScoped;
 
 @TransactionScoped
 public class TransactionScopedBean {
```

Comment:
Could you give this a different name? You're adding two classes with this name; I understand it's not a problem as they are in a different package, but it's a bit unpractical to have two around.